packages/request-pre/src/config.ts

```
@@ -0,0 +1,79 @@
+import { ApiSchemaConfig, ApiSchema } from './api';
+import { requestReturn } from './util/request';
+export const SKIP_NEXT = {};
+export const SKIP_REQUEST = {};
+export type apiConfig = Map<string, Function | {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)